README.md

```diff
@@ -16,6 +16,25 @@ To get `help` command, run the following code:
 
 ## Commands
 
+To create and close milestone:
+```
+bot create milestone title myorg/myrepo
```

Comment:
Think it would be better if you move `myorg/myrepo` before title, so you could be flexible choosing the title

Reply:
Or you can detect repo with regex

Reply:
@zelima I want to keep it in this way, since we have another command `bot create milestone title`, I just wanted them to be in the same convention